video_xblock/tests/unit/test_mixins.py

```diff
@@ -355,6 +357,62 @@ def test_srt_to_vtt(self, convert_caps_to_vtt_mock, requests_mock):
         self.assertEqual(vtt_response.text, 'vtt transcripts')
         convert_caps_to_vtt_mock.assert_called_once_with(text_mock)
 
+    @patch.object(VideoXBlock, 'create_transcript_file')
+    @patch.object(VideoXBlock, 'get_player')
+    @patch('video_xblock.constants.TPMApiLanguage')
+    @patch('video_xblock.constants.TPMApiTranscriptFormatID')
+    @patch('video_xblock.mixins.requests.get')
+    def test_get_translations_from_3playmedia(self, request_get_mock, format_id_mock, lang_data_mock, player_mock,
+                                              create_transcript_file_mock):
+        # Arrange
+        vtt_file_mock = Mock(return_value='vtt_file_text')
+        domain = "https://static.3playmedia.com/"
+        all_transcripts_url = domain + 'files/758521/transcripts?apikey=test_api_key'
+        vtt_translation_url = domain + 'files/758521/transcripts/1653424?apikey=test_api_key&format_id=51'
+
+        transcripts_response_stub = ResponseStub(body=[{
+            "id": 1653424,
+            "media_file_id": 758521,
+            "language_id": 1,
+            "language_name": "English",
+            "type": "TranscribedTranscript"
+        }])
+        vtt_file_response_stub = ResponseStub(body=vtt_file_mock.return_value)
+        request_get_mock.side_effect = [transcripts_response_stub, vtt_file_response_stub]
+
+        format_id_mock.WEBVTT.return_value = 51
+        lang_data_mock.configure_mock(**{
```

Comment:
It looks like here you don't actually need a mock. Because you neither pass it to function under test, nor verify if it's been called.

But mocks are implying additional semantics and are misleading in this case.

So plain object/dict or just a set of vars is enough.